### ISSUE TITLE
[export * from 'xxxx'] Export statement no-duplicate configuration does not working

### ISSUE THREAD

**Issue by euwei7** [antfu/eslint-config#310]
### Describe the bug

Test code:
```ts
// index.ts 
// No error reported
export * from './button'
export * from './button'
export * from './button'
export * from './button'
export * from './button'
```

```js
// eslint.config.js
import antfu from '@antfu/eslint-config'

export default antfu({
  files: ['**/*.ts'],
  rules: {
    'no-duplicate-imports': 'error', 
    'import/no-duplicates': 'error',
  },
})
```

### Reproduction

Test code

### System Info

```Shell
macbook m1 os13
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by antfu** [comment]
We temporarily closed this due to the lack of enough information. We could not identify whether it was a bug or a userland misconfiguration with the given info.
Please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to reopen the issue.
Thanks.

> [Why reproduction is required](https://antfu.me/posts/why-reproductions-are-required)
